xsd-cli/src/main/resources/cli-docs/xsd-help.help

```diff
@@ -22,3 +22,7 @@ EXAMPLE
        Generate types from an XSD schema with a specified relative path of the output file. The output will be saved 
        to the 'custom' submodule in the Ballerina project.
            $ bal xsd schema.xsd --module custom
+
+       Generate types for all the XSD files in a directory. The output will be saved to the types.bal file
+       in the same directory where the command is executed.
```

Comment:
```suggestion
       Generate types for all XSD files in a directory. The types.bal file for each XSD file in the directory is generated within the default package.
```

Reply:
it is incorrect to say `in the same directory where the command is executed`

Reply:
Fixed in [f8f97d4](https://github.com/ballerina-platform/xsd-tools/pull/13/commits/f8f97d43f3a9500cbe031a15a0160ec62dcb9002)